### PR TITLE
Motoko examples phone book

### PR DESCRIPTION
**Pull request by tarek-eg**
closes https://github.com/demergent-labs/azle/issues/63

### CODE REVIEW

File: examples/motoko_examples/phone-book/README.md [hunk 1]
```diff
@@ -1,3 +1,69 @@
 # Phone book
 
-Reimplementation of the [Motoko Phone book example](https://github.com/dfinity/examples/tree/master/motoko/phone-book)
\ No newline at end of file
+This example demonstrates a phone book application that is accessible from your
```
Comment (lastmjs) [inline examples/motoko_examples/phone-book/README.md:3]: Please just link to the original README

File: examples/motoko_examples/phone-book/dfx.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "phone_book": {
+      "type": "custom",
+      "build": "npx azle phone_book",
+      "root": "src",
+      "ts": "src/phone_book/phone_book.ts",
+      "candid": "src/phone_book/phone_book.did",
+      "wasm": "target/wasm32-unknown-unknown/release/phone_book.wasm"
+    }
+  },
+  "www": {
```
Comment (lastmjs) [inline examples/motoko_examples/phone-book/dfx.json:12]: This www object should be within the canisters object

File: examples/motoko_examples/phone-book/README.md [hunk 3]
```diff
@@ -1,3 +1,69 @@
...
+1. Reserve an identifier for your canister.
+
+   ```text
+   dfx canister create --all
+   ```
+
+1. Build your front-end.
+
+   ```text
+   npm install
```
Comment (lastmjs) [inline examples/motoko_examples/phone-book/README.md:48]: Same issues as with the superhoes example probably, try to make everything an exact clone as far as possible, use the asset canister

**Review by lastmjs — CHANGES_REQUESTED** [review]
The problems here are similar to superhoes, probably stemming from the www object in the dfx.json being in the wrong place. If you can fix that and test everything, making sure it's an exact clone as far as possible, and replacing any language in the frontend app that refers to Motoko, then this should be awesome!

**Comment by tarek-eg** [comment]
I think I didn't detect the www location issue because I used the npm start command instead which doesn't require the asset canister to be deployed.

File: .eslintrc [hunk 4]
```diff
@@ -1,3 +1,10 @@
 {
-  "extends": ["prettier"]
-}
\ No newline at end of file
+  "extends": ["prettier"],
```
Comment (lastmjs) [inline .eslintrc:2]: Let's remove this file

**Review by lastmjs — CHANGES_REQUESTED** [review]

**Comment by lastmjs** [comment]
Looking really good, if you've tested it locally from the asset canister and it works, just remove .eslintrc and I can merge this

**Comment by tarek-eg** [comment]
@lastmjs this is tested locally and works fine

**Comment by lastmjs** [comment]
.eslintrc file is still in there

**Comment by tarek-eg** [comment]
> .eslintrc file is still in there

It was originally there, that's the one in the root

**Pull request merged**